frequently-encountered-problems.html

```diff
@@ -248,6 +248,25 @@ <h2>TLS 1.0 compatibility with Node 12 and newer</code></h2>
         </p>
     </dd>
     <br />
+    <dt>
+        <h2>How Tedious handles Unicode datatype conversions when using parameters</code></h2>
+    </dt>
+    <dd>
+        <p>
+            Since all JavaScript strings are Unicode, when using a non-Unicode datatype (such as VarChar), then Tedious will do the conversion 
+            from a Unicode string to a non-Unicode string then send the data to the server. 
+            The conversion may not be what the SQL server expects and cause inaccurate results.
+        </p>
+        <p>
+            There are some alternatives to this. You can use a Unicode datatype as the parameter type. Tedious will send the string as Unicode, 
+            then the SQL server will do the conversion. This will cause a performance hit. 
```

Comment:
> This will cause a performance hit.

I don't think that's always going to be true - we probably should reword this. 😅

Reply:
Good point. I reworded it